resources/js/store/modules/user/actions.js

```diff
@@ -232,6 +232,32 @@ export const actions = {
         });
     },
 
+    /**
+     * Save all tag settings on Tagging.vue
+     */
+    async SAVE_TAGGING_SETTINGS (context)
+    {
+        let title = i18n.t('notifications.success');
+        let body  = i18n.t('notifications.tagging-updated');
```

Comment:
When the variable does not change, use `const`

Reply:
good catch, I fixed them all in `actions.js`